src/main/kotlin/no/nav/familie/ef/iverksett/tilbakekreving/OpprettTilbakekrevingTask.kt

```diff
@@ -116,8 +116,8 @@ class OpprettTilbakekrevingTask(
 
     private fun lagTilbakekrevingRequest(iverksett: IverksettData): OpprettTilbakekrevingRequest {
         // Henter ut på nytt, selv om noe finnes i iverksett-dto'en
-        val enhet = familieIntegrasjonerClient.hentBehandlendeEnhetForBehandling(iverksett.søker.personIdent)!!
-        return iverksett.tilOpprettTilbakekrevingRequest(enhet)
+        val enhet = familieIntegrasjonerClient.hentBehandlendeEnhetForBehandlingMedRelasjoner(iverksett.søker.personIdent).firstOrNull()
```

Comment:
Blir ikke helt klok på hvorfor APIet returnerer en liste, men ser vi kjører `firstOrNull()` i ef-sak.

Reply:
Det var sånn i utgangspunktet - og jeg hadde knekt KS/BA hvis jeg endret til ett objekt i sin tid... Men det er alltid ett objekt i lista her.

Reply:
Vi kunne en periode ha tom liste, men det mener jeg de fikset i NORG for ca. 1,5 år siden.